api/api.go

```diff
@@ -108,6 +111,69 @@ func NewApi(clientset kubernetes.Interface, fasitUrl, clusterDomain, clusterName
 	}
 }
 
+func (api Api) deploy(w http.ResponseWriter, r *http.Request) *appError {
+	requests.With(prometheus.Labels{"path": "deploy"}).Inc()
+
+	deploymentRequest, err := unmarshalDeploymentRequest(r.Body)
+
+	if err != nil {
+		return &appError{err, "unable to unmarshal deployment request", http.StatusBadRequest}
+	}
+
+	//TODO remove this once grace period ends
```

Comment:
Can we specify what we are supposed to remove? Is `ensureProopertyCompatability` deprecated?

Reply:
this as in the line directly beneath